cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -823,13 +827,13 @@ private static CassandraNodeTask getMetadataTask(@NotNull final String executorI
         final Long availableMem = resourceValueLong(headOption(index.get("mem"))).or(0L);
         final Long availableDisk = resourceValueLong(headOption(index.get("disk"))).or(0L);
 
-        if (availableCpus <= cpu) {
-            errors.add(String.format("Not enough cpu resources for role %s. Required %f only %f available.", mesosRole, cpu, availableCpus));
+        if (availableCpus < cpu) {
```

Comment:
Why the change to strictly less than? Won't this now fail if the task needs `2` and the offer has exactly `2`?